chrome/updater/updater_module.cc

```diff
@@ -98,17 +100,24 @@ void Observer::OnEvent(Events event, const std::string& id) {
       status = "Status is unknown.";
     } else if (crx_update_item_.state == ComponentState::kUpToDate &&
                updater_configurator_->GetPreviousUpdaterStatus().compare(
-                   updater_status_string_map.find(UpdaterStatus::kUpdated)
-                       ->second) == 0) {
+                   updater_status_string_map.at(UpdaterStatus::kUpdated)) == 0) {
       status = std::string(
-          updater_status_string_map.find(UpdaterStatus::kUpdated)->second);
+          updater_status_string_map.at(UpdaterStatus::kUpdated));
     } else {
       status = std::string(
           updater_status_string_map.find(status_iterator->second)->second);
     }
     if (crx_update_item_.state == ComponentState::kUpdateError) {
-      status +=
-          ", error code is " + std::to_string(crx_update_item_.error_code);
+      // QUICK_ROLL_FORWARD update, adjust the message to "Update installed,
+      // pending restart"
+      if (crx_update_item_.error_code == static_cast<int>(UpdateCheckError::QUICK_ROLL_FORWARD)) {
+        status = std::string(
+            updater_status_string_map.at(UpdaterStatus::kUpdated));
+      } else {
+        status +=
```

Comment:
Adding the error category was good, but it'd be even better to add a string to each error category and error code. (That's what I meant with "stringification", not just turn an int into a string). This would come at a binary size cost, but that's your tradeoff, since this would only print numbers which then someone would have to convert to human-readable messages (and those messages/enums might change per versions).